src/components/orders/OrderSurplusDisplay/index.tsx

```diff
@@ -101,11 +101,37 @@ const IconWrapper = styled(FontAwesomeIcon)`
   }
 `
 
-export function OrderSurplusTooltipDisplay({ order }: Props): JSX.Element | null {
+const HiddenSection = styled.span<{ showHiddenSection: boolean; strechHiddenSection: boolean }>`
+  display: ${({ showHiddenSection }): string => (showHiddenSection ? 'flex' : 'none')};
+  ${({ strechHiddenSection }): FlattenSimpleInterpolation | false =>
+    strechHiddenSection &&
+    css`
+      width: 3.4rem;
```

Comment:
![image](https://user-images.githubusercontent.com/622217/172732370-dbb77ea9-c956-4f7d-9867-c1a513555ccb.png)
(see the Surplus amount at the right)
Maybe we need to set `width: 100%`; on mobile